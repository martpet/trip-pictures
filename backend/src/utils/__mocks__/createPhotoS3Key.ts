export const createPhotoS3Key = vi
  .fn()
  .mockName('createPhotoS3Key')
  .mockReturnValue('dummyPhotoS3Key');
