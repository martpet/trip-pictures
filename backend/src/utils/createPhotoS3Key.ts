type Props = {
  userId: string;
  fingerprint: string;
};

export function createPhotoS3Key({ userId, fingerprint }: Props) {
  return `${userId}/${fingerprint}`;
}
