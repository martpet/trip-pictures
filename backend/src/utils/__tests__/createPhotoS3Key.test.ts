import { createPhotoS3Key } from '../createPhotoS3Key';

describe('createPhotoS3Key', () => {
  it('return correct value', () => {
    expect(
      createPhotoS3Key({ userId: 'dummyUserId', fingerprint: 'dummyFingerprint' })
    ).toMatchSnapshot();
  });
});
