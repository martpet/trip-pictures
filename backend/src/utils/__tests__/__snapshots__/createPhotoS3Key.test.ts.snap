// Vitest Snapshot v1

exports[`createPhotoS3Key > return correct value 1`] = `"dummyUserId/dummyFingerprint"`;
